wgsl/index.bs

```diff
@@ -90,7 +90,89 @@ that run on the GPU.
  * Features and semantics are exactly the ones of SPIR-V
  * Each item in this spec *must* provide the mapping to SPIR-V for the construct
 
-## Technical Overview TODO ## {#technical-overview}
+## Technical Overview ## {#technical-overview}
+
+WebGPU issues a unit of work to the GPU in the form of a <dfn noexport>provoking command</dfn>.
+[SHORTNAME] is concerned with two kinds of provoking commands:
```

Comment:
Maybe command submission is better in terms of being closer to the API language. Then use just command afterwards.

Reply:
I forgot to address this feedback.

I wanted to stay away from talking about a command being created, scheduled, eventually unblocked, and the actually launched.   (e.g. compare the [6-state progression](https://www.khronos.org/registry/OpenCL/specs/3.0-unified/html/OpenCL_API.html#profiled-states-image) of a kernel in OpenCL 2.0 and later. )

I used "provoking command" to bypass all that conceptual baggage and to assume that a command is starting to be acted on.
I used "provoking" in analogy to "provoking vertex", even if the analogy is imperfect.

Reply:
Looking at the WebGPU spec, there is a defined term "GPU command" we could link to.  Then I would not have to say "provoking".

Reply:
Now it reads like this:

> WebGPU issues a unit of work to the GPU in the form of a [GPU command](https://gpuweb.github.io/gpuweb/#gpu-command). WGSL is concerned with two kinds of GPU commands: